commons/src/test/java/org/eclipse/kapua/commons/event/ServiceEventClientConfigurationTest.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.commons.event;
+
+import org.eclipse.kapua.event.ServiceEventBusListener;
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(JUnitTests.class)
+public class ServiceEventClientConfigurationTest extends Assert {
+
+    ServiceEventBusListener serviceEventBusListener;
+
+    @Test
+    public void constructorRegularTest() {
+        ServiceEventClientConfiguration serviceEventClientConfiguration = new ServiceEventClientConfiguration("address", "subscriberName", serviceEventBusListener);
+        assertEquals("not_equals", "address", serviceEventClientConfiguration.getAddress());
+        assertEquals("not_equals", "subscriberName", serviceEventClientConfiguration.getClientName());
+        assertEquals("not_equals", serviceEventBusListener, serviceEventClientConfiguration.getEventListener());
```

Comment:
Please change the message to: "Expected and Actual values not equal" or something similar. "not_equals" is not a valid comment.